src/main/java/org/hiero/sketch/table/PartialMembershipDense.java

```diff
@@ -0,0 +1,103 @@
+package org.hiero.sketch.table;
+
+import org.hiero.sketch.table.api.IMembershipSet;
+import org.hiero.sketch.table.api.IRowIterator;
+import org.scalactic.exceptions.NullArgumentException;
+
+import java.util.*;
+import java.util.function.Predicate;
+
+
+/**
+ * Created by uwieder on 10/20/16.
```

Comment:
We don't want this comment; you can turn this off if you hover over the comment and click the popup balloon that shows up - it contains code templates.